### PR TITLE
Wrapper for vcf2maf converter

### PR DESCRIPTION
**Pull request by DM2zzion**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [X] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/vcf2maf/vcf2maf.xml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	<tests>
+		<test expect_num_outputs="1">
+			<param name="input1" dbkey="hg19" value="input_test1.vcf" ftype="vcf" />
+			<param name="ref_source" value="history" />
+			<param name="ref" dbkey="hg19" value="test1.fa" ftype="fasta" />
+			<param name="annotation_cache.source" value="no_vep" />
+			<output name="output1" file="output_test1.tabular" ftype="tabular" />
+		</test>
+		<test expect_num_outputs="1">
+			<param name="input1" dbkey="dm6" value="input_test2.vcf" ftype="vcf" />
```
Comment (bgruening) [inline tools/vcf2maf/vcf2maf.xml:132]: a test using the fasta_indexes would be nice ...

**Review by bgruening — COMMENTED** [review]
Very cool! Thanks @DM2zzion this is high quality! Do you need help with the failing test?

**Comment by bgruening** [comment]
> fails because it cannot write the new file $input_name.split.vcf. I suppose on the test system, the tool does not have write access in that case?

Yes the input file directory is read-only on many systems. And everything should happen in a working directory - also for security reasons.

Please try symlinking the input file into the CWD. Something along those lines:

`ln -s '$input' ./foo.vcf && vcf2mat ./foo.vcf ...`

**Comment by DM2zzion** [comment]
> > fails because it cannot write the new file $input_name.split.vcf. I suppose on the test system, the tool does not have write access in that case?
> 
> Yes the input file directory is read-only on many systems. And everything should happen in a working directory - also for security reasons.
> 
> Please try symlinking the input file into the CWD. Something along those lines:
> 
> `ln -s '$input' ./foo.vcf && vcf2mat ./foo.vcf ...`

Thanks for your help! With the last commit I introduced symlinks for the input and reference fasta. Furthermore, I added a test case for fasta_indexes as requested by you. I think the wrapper is ready now :)

**Review by bgruening — APPROVED** [review]
Cool, very nice! Thanks a lot @DM2zzion

**Pull request merged**